ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -374,7 +374,7 @@ def fetch_views(
         """
         for view_name in inspector.get_view_names(schema):
             try:
-                if self.service_connection.scheme == "bigquery":
+                if self.service_connection.scheme.value == "bigquery":
```

Comment:
lets not introduce this kind of checks in sql_source. Instead override this in bigquery connector

Reply:
Okay I'll override this in the bigquery connector